data/campaigns/Winds_of_Fate/scenarios/03x_Advice.cfg

```diff
@@ -0,0 +1,128 @@
+#textdomain wesnoth-wof
+
+[scenario]
+    id=03x_Advice
+    name= _ "Advice"
+    next_scenario=04_Islands
+    {NO_MAP_DATA}
+    carryover_percentage=0
+
+    {INTRO_AND_SCENARIO_MUSIC heroes_rite.ogg ()}
+
+    [story]
+        [part]
+            background={GORLACK_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Gorlack")} + _ "Dominant Vladnir, I can not remain in this land as your ally.
+
+I seek your forgiveness."
+        [/part]
+        [part]
+            background={VLADNIR_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Vladnir")} + _ "There is nothing for me to forgive.
+
+The Spoken Memory of the Glider Caste has pointed your course.
+There is no better heading I may counsel."
+        [/part]
+        [part]
+            background={GORLACK_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Gorlack")} + _ "... I never suspected you held the gliders in such esteem.
+That you had heard the words of the Spoken Memory.
+
+Perhaps glider wisdom overlaps the many written secrets your own caste wards."
+        [/part]
+        [part]
+            background={VLADNIR_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Vladnir")} + _ "I am forbidden to tell.
+The Gliders may share their Memories.
+We Clashers must keep our Secrets.
+
+Yet I may say this...
+
+Your glider first intendant is well chosen for the journey ahead of you."
+        [/part]
+        [part]
+            background={VLADNIR_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Vladnir")} + _ "Many matters are now in motion. I go now, for there are preparations to be made...
+
+<i>May your fire light the way in darkness.</i>"
+        [/part]
+        [part]
+            background={GORLACK_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Gorlack")} + _ "<i>May your ancestors tread beside you.</i>"
+        [/part]
+        [part]
+            background={RESHAN_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Reshan")} + _ "My Dominant Gorlack, your Flight has assembled upon the eastern landing."
+        [/part]
+        [part]
+            background={GORLACK_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Gorlack")} + _ "There is no need for you to address me by status, Reshan.
+We are still equals regardless of rank."
+        [/part]
+        [part]
+            background={RESHAN_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Reshan")} + _ "My friend, we can not be equals, for I owe you my life.
```

Comment:
'can not' should be cannot

Reply:
Will change to:

"My friend, we are not equals, for I owe you my life."

Reply:
change applied